firestore/src/android/server_timestamp_behavior_android.h

```diff
@@ -1,3 +1,5 @@
+// Copyright 2020 Google LLC
+
 #ifndef FIREBASE_FIRESTORE_CLIENT_CPP_SRC_ANDROID_SERVER_TIMESTAMP_BEHAVIOR_ANDROID_H_
 #define FIREBASE_FIRESTORE_CLIENT_CPP_SRC_ANDROID_SERVER_TIMESTAMP_BEHAVIOR_ANDROID_H_
 
```

Comment:
Lint: `Lines should be <= 80 characters long `<hidden value="cpplint-file-comment"></hidden>